typescript/src/schema/proto.ts

```diff
@@ -288,6 +289,23 @@ function toLRInterface(type: string, inputType: string) {
   return inputType.replace(/\.([^.]+)$/, '.I' + type);
 }
 
+function getHeaderParms(rule: plugin.google.api.IHttpRule): string {
+  const message = rule.post
+    ? rule.post
+    : rule.delete
+    ? rule.delete
+    : rule.get
+    ? rule.get
+    : rule.put
+    ? rule.put
+    : rule.patch;
+  if (message){ 
+    const res = message.match('(?<={)(.*)(?==)');
```

Comment:
Positive look-behinds are very rarely used. If you just need to grab something in the curly brackets, just do `/{(.*?)=/`.

Reply:
got it, `/{(.*?)=/` seems to include `=`, use `/{(.*?)(?==)/` instead.